applications/osb-portal/src/components/workspace/NewWorkspaceToolBox.tsx

```diff
@@ -67,8 +67,9 @@ export const WorkspaceToolBox = (props: any) => {
           </Grid>
           <Grid item={true} xs={12} sm={12} md={8} lg={9}>
             <Grid container={true} direction="row" justify="space-between" spacing={5} >
-              <Grid item={true} xs={12} sm={3} className={classes.justifyCenter} >
+              <Grid id='computational-modeling-item' item={true} xs={12} sm={3} className={classes.justifyCenter} >
```

Comment:
here a class would be preferable as the component is potentially reusable. But it's not, so can be fine